data/org.freedesktop.impl.portal.Screenshot.xml

```diff
@@ -49,7 +49,9 @@
             <term>interactive b</term>
             <listitem><para>
               Hint whether the dialog should offer customization before taking a screenshot.
-              Defaults to no.
+              Defaults to no. Since version 3 of #org.freedesktop.portal.Screenshot,
+              non-interactive screenshots are mediated by the permission store, so
+              implementations should not to show any additional interaction.
```

Comment:
```suggestion
              Defaults to no. Since version 3 of #org.freedesktop.portal.Screenshot,
              non-interactive screenshots are mediated by the permission store, so
              implementations should not show any additional interaction.
```

Reply:
Good catch